chart/flux/values.yaml

```diff
@@ -201,15 +205,14 @@ memcached:
   nodeSelector: {}
   tolerations: []
   affinity: {}
-  resources: {}
-    # If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  resources:
+    # If you do want to specify resource limits, uncomment the following and adjust values
     # limits:
     #  cpu: 100m
     #  memory: 628Mi
-    # requests:
-    #  cpu: 50m
-    #  memory: 512Mi
+    requests:
+     cpu: 50m
+     memory: 512Mi
```

Comment:
I would set it to 64Mi as most users will never get to 512Mi unless you're using hundreds of unique images.